infrastructure/base/modules/env/main.tf

```diff
@@ -391,6 +391,21 @@ module "purge_users_cron" {
   service_account_email = module.jobs_cloudrun.service_account_email
 }
 
+module "frontend_uptime_check" {
+  source = "../uptime-check"
+  name   = "${var.project_name} Frontend"
+  host   = var.domain
+  email  = var.uptime_alert_email
+}
+
+module "api_uptime_check" {
+  source = "../uptime-check"
+  name   = "${var.project_name} API"
+  host   = var.domain
+  path   = "/backend/health_check"
```

Comment:
I understand we check BE availability by using the health endpoint, but I don't understand how we check FE, i.e. there's no path given in the FE uptime check?

Reply:
correct, there is no path, we check if the homepage is up

Reply:
🧙